4918 telegraf write example (#4929)
influxdata/docs-v2: content/influxdb/cloud-dedicated/write-data/use-telegraf/configure/_index.md

Author: @sanderson Please review this. Moved manual-config -> configure/_index.md

Reviewer: Looks great!

---
title: Configure Telegraf for InfluxDB
description: >
  Telegraf is a plugin-based agent with plugins are that enabled and configured in
  your Telegraf configuration file (`telegraf.conf`).
  Learn how to create Telegraf configuration files that work with InfluxDB.
menu:
  influxdb_cloud_dedicated:
    name: Configure Telegraf
    parent: Use Telegraf
weight: 101
---

Telegraf is a plugin-based agent with plugins are that enabled and configured in
your Telegraf configuration file (`telegraf.conf`).

> The following are options you have for creating Telegraf configuration files

Author: ```suggestion
The following options are available for creating Telegraf configuration files
```

> The following are options you have for creating Telegraf configuration files

Author: ```suggestion
The following options are available for creating Telegraf configuration files
```

that work with InfluxDB {{< current-version >}}.

> {{< children >}}

Reviewer: I don't think InfluxDB Cloud Dedicated is going to provide Telegraf configuration storage or a Telegraf builder. We can probably replace `{{< children >}}` with the contents of "Manually configure Telegraf" and remove that doc altogether.
